napari/viewer.py

```diff
@@ -43,6 +50,7 @@ def __init__(
         order=(),
         axis_labels=(),
         show=True,
+        console_local_update=True,
```

Comment:
I don't love expanding the Viewer API here. I would either hardcode it as true (we can revert if people start running into issues), or do as @sofroniewn suggests and push this all to the napari-console plugin. (I do think that is the proper place for this.)

Reply:
yeah, maybe we can just hard code this as true. if people really object we can add a way to disable it